src/main/java/com/github/vkorobkov/jfixtures/loader/FixtureValue.java

```diff
@@ -24,6 +26,11 @@ public FixtureValue(Object value) {
         }
     }
 
+    @XmlValue
```

Comment:
Any reason not to serialize the `type` field?

Reply:
Do you mean
`    @XmlValue
    private final Object value;`